android/build.gradle

```diff
@@ -25,14 +25,14 @@ android {
         applicationId 'io.urho3d'
         versionCode 1
         versionName '1.0.0'
-        minSdkVersion 17
+        minSdkVersion 28
         targetSdkVersion 33
         ndk {
             abiFilters 'armeabi-v7a', 'arm64-v8a', 'x86_64'
         }
         externalNativeBuild {
             cmake {
-                arguments '-DANDROID_STL=c++_static', '-DANDROID_PLATFORM=android-21', '-DANDROID=1',
+                arguments '-DANDROID_STL=c++_static', '-DANDROID_PLATFORM=android-28', '-DANDROID=1',
```

Comment:
This should probably reflected in documentation somewhere. So we going from Android 5.0 (99.6% of devices) to Android 9.0 (91.7% of devices)? Is it absolutely necessary?

Reply:
IMO this is really not that important. Users are expected to copy `android` dir and thus `build.gradle` and adapt them to their own needs. They can use any version they like.